client/store/state.js

```diff
@@ -4,4 +4,6 @@ export default {
     username: undefined,
     email: undefined,
     pk: undefined,
+
+    error_msg: {},
```

Comment:
I think you should make something to null this property every time user enter or leave a view. Otherwise you may pass errors between views e.g if user fail password in login and then he change view to register he will get password error instantly

Reply:
I think you can use Global After / Before hooks for that